### PR TITLE
Duplicate detector : Show how many items are left in main page

### PR DESCRIPTION
**Pull request by plive498**
**Implements Feature:** Show how many items are left in main page of a duplicate detector.

<br />

Summary of changes in this PR:

- Show how many items are left in main page of a duplicate detector.

Additional commit notes for project team:

- `DuplicateDetectorActivity.updateTitle` has parameter `count`
if count>0, `updateTitle` update title to "n duplicates" for the detail page
if count<0, `updateTitle` update title to "n item(s) left" for the main page
if count=0, `updateTitle` update title to "Duplicate Detector" for the main page without any items

<br />
@AVnetWS/admin-team

### CODE REVIEW

File: app/src/main/java/me/devsaki/hentoid/activities/DuplicateDetectorActivity.kt [hunk 1]
```diff
@@ -133,10 +133,14 @@ class DuplicateDetectorActivity : BaseActivity() {
     }
 
     fun updateTitle(count: Int) {
-        binding!!.toolbar.title = if (count > -1) resources.getQuantityString(
+        binding!!.toolbar.title = if (count > 0) resources.getQuantityString(
```
Comment (RobbWatershed) [inline app/src/main/java/me/devsaki/hentoid/activities/DuplicateDetectorActivity.kt:136]: Using `abs` (https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.math/abs.html) should simplify that

**Comment by RobbWatershed** [comment]
Thanks for your contribution plive 👍 

I see two things that can be improved there :
- The `<0  => main page` / `>0 => details page` convention will probably be a little difficult to understand a few months later if the only explanation is in this PR.

Could you add comments as why you pass negative values to `updateTitle` ?

- Calling the ViewModel directly to get values (`DuplicateDetectorActivity:79`) isn't exactly the right way to use the MVVM pattern. Ideally, you should _observe_ values in the Activity or Fragment and react to their changes that way.
e.g. `DuplicateMainFragment:126`
``` kotlin
 // Observe what the ViewModel publishes; process it using onDuplicatesChanged
viewModel.allDuplicates.observe(viewLifecycleOwner, { this.onDuplicatesChanged(it) })
```
Could you modify `DuplicateDetectorActivity` to use that technique instead of calling `viewModel.allDuplicates.value` directly ?

**Comment by plive498** [comment]
Thanks for your review. I added new commits.

**Comment by RobbWatershed** [comment]
Looks all good !

**Pull request merged**